packages/core/components/StatusMessage/StatusMessage.module.css

```diff
@@ -4,7 +4,7 @@
     width: 40vw; 
     height: 100%;
     left: 30px;
-    z-index: 1000;
+    z-index: 99999999999;
```

Comment:
originally didn't do this because then it overlaps with the modals, but realize now it's not really an issue since they can just be closed 
<img width="1287" alt="image" src="https://github.com/user-attachments/assets/57e40560-5915-49b2-b3ba-e276f5d79637" />